types/ckeditor/index.d.ts

```diff
@@ -8,7 +8,7 @@
 // See https://github.com/DefinitelyTyped/DefinitelyTyped/issues/1827 for more informations.
 
 interface Window {
-    CKEDITOR_BASEPATH: string;
+    CKEDITOR_BASEPATH?: string;
```

Comment:
Note that this indicates that the property may not be present, whereas `CKEDITOR_BASEPATH: string | undefined` would indicate that it must be present but may have the value `undefined`.  I'm not sure which is correct for this library.

Reply:
The property may not be present as it has to be set by the user if needed: [CKEDITOR_BASEPATH](https://ckeditor.com/docs/ckeditor4/latest/guide/dev_basepath.html#solution-ckeditor_basepath).
CKEditor never set the property. It only reads it, see [core/ckeditor_base.js](https://github.com/ckeditor/ckeditor-dev/blob/a49f7cb3426b3fde346aca07fa7a514959c9854e/core/ckeditor_base.js#L109-L151).

Also, after setting-up a simple CKEditor, we have
```javascript
'CKEDITOR' in window // => true
'CKEDITOR_BASEPATH' in window // => false
```